sync_config.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+cd /srv/runtime_data/current
+
+while true
```

Comment:
Is this actually being consistent now?

Reply:
it actually always was....it was just the aws console being a POS.....FYI the "refresh" button on the ECS task logs screen does not actually refresh or pull the latest logs....